### PR TITLE
Added XTONE Smart Stomp and more docs

### PR DESCRIPTION
**Pull request by galkk**
Added settings for XTONE Smart Stomp and a bit more links to configs/examples. I feel that it will be beneficial for new users to see working examples.

I plan to check my other devices, if they're working: Yamaha THR10 and MOOER GE300, but, most likely, I'll create another PRs for them. Thank you for your great work.

### CODE REVIEW

**Comment by galkk** [comment]
You can see how it looks like in my repo start page https://github.com/galkk/rs_asio#known-issues

**Comment by mdias** [comment]
This is so much needed! Thanks for the effort!
I'll make a suggestion before moving; since we're bound to want to make more of these for different interfaces, this would soon pollute the main readme file, so what I would suggest is having a folder per interface on `docs/` and the guide in there.

For example, for this interface we'd have `docs/xtone-smart-stomp/GUIDE.md` and we'd link to that from the interface list on the main readme. This would make it easy and clean to support many different interfaces. What do you think?

**Comment by galkk** [comment]
> I'll make a suggestion before merging; since we're bound to want to make more of these for different interfaces, this would soon pollute the main readme file, so what I would suggest is having a folder per interface on docs/ and the guide in there.

That's actually how I was thinking to do that in the beginning, but thought that this may be an overkill :)

Moved specific interface doc into docs/interface_name and added README.md there to have it rendered by github automatically

File: README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ It patches game code at runtime to allow intervening in the process of WASAPI de
 ## How to use
 
 - Copy the contents (`avrt.dll`, `RS_ASIO.dll`, `RS_ASIO.ini`) of [latest release](https://github.com/mdias/rs_asio/releases/latest) (zip arhive release-xxx.zip) to the game folder.
+  - If you're using Steam version of Rocksmith, you can find local folder by right clicking on a Rocksmith in your library, and selecting menu "Manage" -> "Browse local files"
```
Comment (mdias) [inline README.md]: We're currently only compatible with Steam version, can you please change the wording to avoid people thinking otherwise?

**Comment by mdias** [comment]
I'm very happy with this!
Please tell me when you're done so I can merge :)

File: README.md (hunk 1 above)
Comment (galkk) [reply README.md]: done

**Comment by galkk** [comment]
Fixed steam-related wording

**Comment by mdias** [comment]
Merged! Thanks a lot for the contribution 👍

**Pull request merged**